### ISSUE TITLE
[GBC] Megamen Xtreme white line on display

### ISSUE THREAD

**Issue by puggile** [OtherCrashOverride/go-play#49]
When you play this rom ( i tried USA version ) you get big white lines on screen before ( I think ) loading a new scene.
You can reproduce this simply starting a new game and moving on right for a little while.

**Comment by OtherCrashOverride** [comment]
This is correct operation.  The LCD is being disabled which results in a white screen on a real gameboy.
https://www.youtube.com/watch?v=adHAuvRkhPU&t=346
